app/models/user.rb

```diff
@@ -1,14 +1,14 @@
 class User < ActiveRecord::Base
-  has_secure_password
+  has_secure_password validations: false
 
   has_many :stems, class_name: "Track", foreign_key: "creator_id"
   has_many :remixes, class_name: "Track", foreign_key: "creator_id"
   has_many :circles, foreign_key: "creator_id"
 
   validates_presence_of :user_name
-  validates_presence_of :email
+  validates_presence_of :email, unless: "sc_token.present?"
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.